pkg/config/validation/virtualservice.go

```diff
@@ -141,6 +141,13 @@ func validateHTTPRouteMatchRequest(http *networking.HTTPRoute, routeType HTTPRou
 					if header == nil {
 						errs = appendErrors(errs, fmt.Errorf("header match %v cannot be null", name))
 					}
+
+					if _, ok := header.GetMatchType().(*networking.StringMatch_Prefix); ok {
+						if header.GetPrefix() == "" {
+							errs = appendErrors(errs, fmt.Errorf("header prefix match %v must have a value", name))
```

Comment:
"can not be empty" or "must be at least 1 character" looks better.

Reply:
`can/must/may not be empty` matches the wording for similar empty string checks in our validation

Reply:
Updated!